pages/cmake.md

```diff
@@ -0,0 +1,30 @@
+# cmake
+
+> CMake is a cross-platform  build system generator. Projects specify
+       their build process with platform-independent CMake listfiles included
+       in each directory of a source tree with the name CMakeLists.txt. Users
+       build a project by using CMake to generate a build system for a native
+       tool on their platform.
```

Comment:
This description is waaaay too long :) Please follow the [page format guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).